.github/workflows/docsbuild-release.yaml

```
@@ -0,0 +1,73 @@
+name: ansible-release-porting-guide
+on:
+  workflow_dispatch:
+    inputs:
+      ansible_version:
+        description: 'Release Version. Example : 11.1.0'
+        required: true
+      ansible_major_version:
+        description: 'Exmaple 11'
+        required: true
+
+env:
+  CI_COMMIT_MESSAGE: 'Add Ansible community "${{ inputs.ansible_version }}" porting guide'
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  CI_PR_BODY: |
+
+      ##### SUMMARY
+
+      ##### ISSUE TYPE
+
+      -   Docs Pull Request
+
+      ##### COMPONENT NAME
+
+      docs/docsite/rst/porting_guides/'porting_guide_"${{ inputs.ansible_major_version }}".rst'
+
+jobs:
+  build:
+    name: 'Build Ansible community distribution (${{ inputs.ansible_version }})'
+    runs-on: ubuntu-latest
+
+    steps:
+
+# checkout the /ansible/ansible-documentation & ansible-build-data
+      - name: Check out ansible-documentation
+        uses: actions/checkout@v3
+        with:
+          path: ansible-documentation
+          ref: devel
+
+      - name: Checking out to a new branch & setting the user details
+        run: |
+          pwd
+          ls
+          cd ansible-documentation
+          git checkout -b "publish-${{ inputs.ansible_version }}"
+          git config --global user.name "${{github.actor}}"
+          git config --global user.email "${{github.actor}}@users.noreply.github.com"
+
+# Get the Porting guide from the tarball
+
+      - name: Commit ansible-documentation
+        run: |
+          wget https://files.pythonhosted.org/packages/source/a/ansible/"ansible-${{ inputs.ansible_version }}".tar.gz
+          tar -xzf "ansible-${{ inputs.ansible_version }}".tar.gz
+          cp "ansible-${{ inputs.ansible_version }}"/"porting_guide_${{ inputs.ansible_major_version }}.rst" ./ansible-documentation/docs/docsite/rst/porting_guides/
+
+# the actual steps for commit
+
+      - name: Push the changes to ansible-documentation github repo
+        run: |
+         cd ./ansible-documentation
+         git add ./docs/docsite/rst/porting_guides/"porting_guide_${{ inputs.ansible_major_version }}.rst"
+         git commit -m "${{ env.CI_COMMIT_MESSAGE }}"
+         git push origin "publish-${{ inputs.ansible_version }}"
+
+      - name: Create PR to the ansible-documentation
+        run: |
+          cd ansible-documentation
+          gh pr create -B devel -H "publish-${{ inputs.ansible_version }}" --title "${{ env.CI_COMMIT_MESSAGE }}"  --body "${{ env.CI_PR_BODY }}"
```

```suggestion
          gh pr create -B devel -H 'publish-${{ inputs.ansible_version }}' --title '${{ env.CI_COMMIT_MESSAGE }}'  --body '${{ env.CI_PR_BODY }}'
```

Although, I'd probably use a `--body-file` option and feed it a pre-written file since it's much easier to pass and escape a path as a CLI option.

---

` -body-file`  goes to my read list.